workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.test.ts

```diff
@@ -287,24 +288,74 @@ describe('DynamicAuthorizationService', () => {
   });
 
   describe('addUserToGroup', () => {
-    it('returns userID and groupID when user was successfully added to the group', async () => {
-      mockGroupManagementPlugin.addUserToGroup = jest.fn().mockResolvedValue({ data: { userId, groupId } });
+    let auditServiceWriteSpy: jest.SpyInstance;
+    let actor: object;
+    let source: object;
+    let action: string;
 
-      const { data } = await dynamicAuthzService.addUserToGroup({
-        groupId,
-        userId,
+    beforeAll(() => {
+      auditServiceWriteSpy = jest.spyOn(auditService, 'write');
+
+      action = 'addUserToGroup';
+      actor = mockUser;
+      source = {
+        serviceName: 'DynamicAuthorizationService'
+      };
+    });
+
+    afterEach(jest.resetAllMocks);
+
+    it('returns userID and groupID when user was successfully added to group', async () => {
+      const mockReturnValue = { data: { userId: 'userId', groupId: 'groupId' } };
```

Comment:
`userId` and `groupId` are described at the top level `describe` block. Lets use those values here.